ui/src/app/base/sagas/websockets.ts

```diff
@@ -71,7 +73,13 @@ export const fileContextRequests = new Map<
 >();
 
 // A store of websocket endpoints that are being polled.
-const pollingRequests = new Set<WebSocketEndpoint>();
+const pollingRequests = new Set<PollRequestId>();
+
+const pollRequestId = (action: WebSocketAction | AnyAction): PollRequestId => {
+  const endpoint = `${action.meta?.model}.${action.meta?.method}`;
```

Comment:
Polling used to be for a single endpoint as there's no point having multiple polls for the same request, however, for checking images we need to support multiple polling events as each check_images request includes a list of controller ids to check the images for.